localenv/mock-account-servicing-entity/Dockerfile

```diff
@@ -1,21 +1,59 @@
-FROM node:18.13.0-slim as builder
+FROM node:18-alpine AS base
```

Comment:
There's an approximately 80MB difference between the `slim` and `alpine images.. While researching, I stumbled upon [distroless](https://github.com/GoogleContainerTools/distroless). I'll dive deeper into these image options. Feel free to share any thoughts on which image type we should opt for.

Reply:
It looks like the `distroless/nodejs:18` docker image has some critical vulnerabilities:
<img width="576" alt="image" src="https://github.com/interledger/rafiki/assets/60383558/785905db-5fa4-4be8-b202-690373e7eda4">

The `distroless` images are also maintained by Google (I assume 🤔). There is a 15-17MB difference between `distroless` and `alpine`.

Reply:
I'm not too keen on distroless because I like to be able to access the shell, which distroless does not have. Although I guess if that's the only concern it could probably be added like so: https://stackoverflow.com/a/71756170/12893676